HW_Task1.py

```diff
@@ -0,0 +1,53 @@
+# Задача 1. Напишите программу вычисления 
+# арифметического выражения заданного строкой. 
+# Используйте операции +,-,/,*. 
+# Приоритет операций стандартный.
+# Пример:
+# 2+2 => 4;
+# 1+2*3 => 7;
+# 1-2*3 => -5;
+
+# Добавьте возможность использования скобок, меняющих приоритет операций.
+# Пример:
+# 1+2*3 => 7;
+# (1+2)*3 => 9;
+
+import re
+
+expr = '1+2*3+(5-3)/2+11'
+s = re.findall(r'\d+|[()+\-*\/]', expr)
+
+print(f'\n{expr} = ?')
+
+def my_calc(s):
+    calc = {'*': lambda x, y: x * y, '/': lambda x, y: x / y, '+': lambda x, y: x + y, '-': lambda x, y: x - y}
```

Comment:
```suggestion
    calc = {
        '*': lambda x, y: x * y,
        '/': lambda x, y: x / y,
        '+': lambda x, y: x + y,
        '-': lambda x, y: x - y,
    }
```
Такая запись будет более понятна.